unit-tests/live/dfu/test-device-fw-compatibility.py

```diff
@@ -14,10 +14,12 @@
 d400_fw_min_version_2 = 'Signed_Image_UVC_5_12_7_100.bin'
 d400_fw_min_version_3 = 'Signed_Image_UVC_5_12_12_100.bin'
 d400_fw_min_version_4 = 'Signed_Image_UVC_5_13_0_50.bin'
+d400_fw_min_version_5 = 'D4XX_FW_Image-5.17.0.9.bin'
```

Comment:
Lets align the name to other names please:
e.g.
`'Signed_Image_UVC_5_17_0_9.bin'`